components/Dashboard/FormPane.tsx

```diff
@@ -3,7 +3,7 @@ import Image from 'next/image'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import { useState } from 'react'
-
+import toast from 'react-hot-toast'
```

Comment:
Move the import statement above as we're separating our imports from built-in libs to custom libs